demo-edit-snapshot.md

```
@@ -0,0 +1,79 @@
+- table [ref=e3]:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)